tools/generate_tables_sql.php

```diff
@@ -52,74 +52,74 @@
             continue;
         }
         switch ($bits[0]) {
-            //generate the CREATE TABLE syntax
+        // generate the CREATE TABLE syntax
         case "table":
-            $filename = "../project/tables_sql/".$bits[1].".sql";
-            $output   = "";
+            $tablename = $bits[1];
+            $filename  = "../project/tables_sql/".$tablename.".sql";
+            $output    = "";
             if (isset($opts["D"])) {
-                $output = "DROP TABLE IF EXISTS `$bits[1]`;\n";
+                $output = "DROP TABLE IF EXISTS `$tablename`;\n";
             }
-            // There's no good way to display this SQL command without exceeding
-            // out line length requirements, so disable phpcs for this chunk.
-            // phpcs:disable
```

Comment:
Adding this file to this PR as the code is pretty much the same:
- remove a phpcs:disable
- consistent spacing around =